ntoskrnl/cc/copy.c

```diff
@@ -400,11 +405,40 @@ CcCanIWrite (
         return FALSE;
     }
 
-    /* Otherwise, if there are no deferred writes yet, start the lazy writer */
-    if (IsListEmpty(&CcDeferredWrites))
+#if DBG
+    /* Output to debug log, before handling the lazy write */
+    DPRINT1("Actively deferring write for: %p \"%wZ\"\n", FileObject, &FileObject->FileName);
+    DPRINT1("Because:\n");
```

Comment:
With 96 MB RAM:

Example output, copying a 400 MB file:
```
(/ntoskrnl/cc/copy.c:410) Actively deferring write for: FF9AF750 "\400MBfileForCore19200.xex"
(/ntoskrnl/cc/copy.c:411) Because:
(/ntoskrnl/cc/copy.c:417)     There are too many cache dirty pages: 0x23c0 + 0x10 >= 0x23cb

(/ntoskrnl/cc/copy.c:410) Actively deferring write for: FF9AF750 "\400MBfileForCore19200.xex"
(/ntoskrnl/cc/copy.c:411) Because:
(/ntoskrnl/cc/copy.c:424)     Available pages are below throttle bottom: 0x41 <= 0x50

(/ntoskrnl/cc/copy.c:410) Actively deferring write for: FF9AF750 "\400MBfileForCore19200.xex"
(/ntoskrnl/cc/copy.c:411) Because:
```